app/models/hackbot/interactions/sql.rb

```diff
@@ -0,0 +1,133 @@
+class Executer
```

Comment:
Perhaps? Wasn't sure the best way to do this, but the logic is very contained to this one file and won't ever be used elsewhere.

Since it won't be used anywhere else, I think `sql.rb` is a good home for it for now.